src/widget.js

```diff
@@ -102,6 +117,44 @@ export default function (base) {
 		static view_name = "AnyView";
 		static view_module = name;
 		static view_module_version = version;
+
+		/** @param {Parameters<InstanceType<base["DOMWidgetModel"]>["initialize"]>} args */
```

Comment:
Ugly hack to make sure we appropriately get the parameter types. Keeping the anywidget src as JS instead of TS makes it so that we don't have to maintain a webpack config on for the `jupyter labextension build` (the default config currently works out of the box).